include/swift/ABI/Task.h

```diff
@@ -259,7 +259,7 @@ struct ResultTypeInfo {
     storeEnumTagSinglePayload(v, whichCase, emptyCases);
   }
   void vw_destroy(OpaqueValue *v) {
-    destroy(v);
+    destroy(v, nullptr);
```

Comment:
Suggestions for how to test this with lit would be appreciated. I reckon we need some mock `SerialExecutor` implementation and `Task` invocation in such test? I've reproduced it with JavaScriptKit's `JavaScriptEventLoop: SerialExecutor` in the first place with [this code](https://github.com/swiftwasm/JavaScriptKit/blob/d3b26d33606a6bd2d416f8af4ba5e4e1d0be3e91/Examples/Basic/Sources/main.swift#L37).

Reply:
Ideally I would love to see runtime tests covering this. Can we / do we have runtime tests running wasm?

Reply:
@MaxDesiatov I think this issue could be reproduced with default cooperative executor.

Reply:
Yes, sounds good to me